codesandbox/.gitignore

```diff
@@ -0,0 +1 @@
+!*
```

Comment:
Is this file needed?

Reply:
I added this one to "unignore" the public/ folder in the examples.